src/pages/hotel/booking/booking-confirmation/hotel-overview/HotelOverview.scss

```diff
@@ -4,14 +4,18 @@
   margin-left: 3px;
   padding: 2px;
 }
+.parent {
```

Comment:
No need to specify the height i think. and the width can be increase little bit also instead of specifying width, we can change it to min-width.
`.book-image-parent {
  min-width: 250px;
}`